doc/meeting_notes.rst

```diff
@@ -0,0 +1,18 @@
+=============
+Meeting notes
+=============
+
+.. item:: MEETING-2019_01_01 Meeting minutes from review meeting 1 January 2019
+
+    In this meeting we go over checklist items and answer their questions. The answers here have priority over the
+    checkbox values in the PR description on GitHub.
+
+.. checkbox-result:: QUE-UNIT_TESTS yes
```

Comment:
aren't these the same items that are already checked in the MR? This directive should check new items so that we see that there is no difference (maybe one, where we prove that this can overwrite the MR one).

Reply:
so this proves it has priority over, but lets add one item which is not in MR, but it is checked here (can have another regex, so that we can add another `item-attributes-matrix` for visualization.